app/lib/system_name.rb

```diff
@@ -16,7 +16,7 @@ def validates_system_name(opts = {})
 
       return unless opts[:uniqueness_scope]
 
-      validates :system_name, uniqueness: opts[:uniqueness_scope] == true ? true : { scope: opts[:uniqueness_scope] }
+      validates :system_name, uniqueness: opts[:uniqueness_scope] == true ? { case_sensitive: true } : { case_sensitive: true, scope: opts[:uniqueness_scope] }
```

Comment:
if we are here, then `uniqueness_scope` must be `true`. Why have the ternary operator here?